geometry_msgs/PoseStamped start
float32 step_size
float32 precision
uint32 max_length
---
nav_msgs/Path path
